src/lopcodes.c

```diff
@@ -102,3 +102,15 @@ LUAI_DDEF const lu_byte luaP_opmodes[NUM_OPCODES] = {
  ,opmode(0, 0, 0, 0, 0, iAx)		/* OP_EXTRAARG */
 };
 
+#ifdef WITH_YK
+/* Is the instruction `i` the start of a loop? */
+int luaP_is_loop_start(Instruction i) {
```

Comment:
Not a killer distinction, but this might be better as a macro.

I also think the `switch` is a long-winded way of saying `return GET_OPCODE(i) == OP_FORLOOP`?

Reply:
For now that's true. I wasn't sure if other opcodes might require analysis to determine if this is the start of a loop.

Reply:
> Not a killer distinction, but this might be better as a macro.

Maybe, but if it got any more complicated than checking a set of opcodes, i would probably prefer a function.

Reply:
Fixed in ade6bbcc585